### PR TITLE
Merging Feature Branch Into Development

### PR DESCRIPTION
**Pull request by Kyle-Law**
Thank you for reviewing my project, it's a basic Restaurant Web App powered by JavaScript. 

- You can access the Live Version [HERE](https://rawcdn.githack.com/Kyle-Law/Restaurant-Page/31105de173296d30a1ebf2d5364fd32dcc7c5fbc/dist/index.html)
- The entire DOM is generated by JavaScript.
- Webpack is setup
- Project Snapshots Here:
image

Looking forward to your review :)

### CODE REVIEW

**Review by rOluochKe — CHANGES_REQUESTED** [review]
Hello @Kyle-Law 
### You have done well so far :clap: However, you have a few issues to fix before your app can be approved.
- [ ] Remove all the commented lines in your codebase
- [ ] Separate the application logic from the dom manipulation
- [ ] In your package.json file, you can add the following lines in the script object. More automation
```
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional

```
- [ ] You can install `webpack-dev-server` and configure properly, this helps with the development environment
- [ ] Use a better font that is easily readable, you search from google fonts online library

### Make all these changes and submit for another code review. Good luck.
- Happy coding!

By Raymond
Slack @iamraymond

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+      navBtn.setAttribute('class', 'nav-btn');
+      navBtn.setAttribute('id', 'home-btn');
+      navBtn.innerText = linkText;
+      navBtn.addEventListener('click', () => {
+        tabController(section);
+      });
+      listItem.appendChild(navBtn);
+      return listItem;
+    };
```
Comment (rOluochKe) [inline src/index.js:57]: ### Separate dom manipulation from the logic

**Comment by Kyle-Law** [comment]
Good day! Thanks @rOluochKe for reviewing my project! Your recommendations have been implemented as followed:
- Unnecessary comments are removed
- more automation lines have been added into scripts tag in package.json
- A more readable font is used.
- Scripts have been modularized into 5 scripts

Live link is updated [HERE](https://rawcdn.githack.com/Kyle-Law/Restaurant-Page/9ad2ecacec36d866bff1cc6e161d7a8829daa8fc/dist/index.html)

Snapshots are updated as well:
image

Thanks and looking forward to another review :)

**Review by Forison — APPROVED** [review]
## Status:: Approved✔️

# 🙋‍♂️ Kyle 

### Good job on addressing the issues raised in the previous review. Go ahead and merge the branch and proceed to the next task🤗.

**[OPTIONAL ]**

- Kindly make it a habit to always add a description to repositories you create on Github.

#### Happy coding👨‍💻

Addo Forison

**Pull request merged**